### PR TITLE
Helm-chart for Altran-v2x

### PR DESCRIPTION
**Pull request by kuldeep2samasi**
Submitting for Review.

### CODE REVIEW

**Comment by kuldeep2samasi** [comment]
Added required License details on files. Please review and approve.

**Review by abjyoti — DISMISSED** [review]

File: applications/altran-v2x/readme.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Separate container images are provided for OBU / RSU apps which can be onboarded (as pods) and communicate over UDP/IP interface. A Human Machine Interface (HMI) app is provided for visualization and demo purpose. In real deployment scenario the UDP/IP interface could be replaced by C-V2X PC5 or C-V2X Uu air interface
+
+Retrieve Container Images and Helm Charts
+
+Containers: Contact Altran Sales
+
+Helm Charts: https://gurftp03.aricent.com/webclient/Login.xhtml
+
+Pre-Requisite:
```
Comment (abjyoti) [inline applications/altran-v2x/readme.md:29]: Which deployment flavor of OpenNESS experience kit is used for this Application Demonstration?

Comment (abjyoti) [reply applications/altran-v2x/readme.md:29]: Please provide the details of the OpenNESS version that is used for verifying this V2X application to ensure the users are having the details of the OpenNESS.

File: applications/altran-v2x/readme.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+Overview of Altran V2X Apps:
+=============================
+
+Altran V2X solution enables safety & non-safety applications for V2V,V2I,V2P and V2N scenarios for Road Side Units (RSU), On Board Units (OBU) and Smart Pedestrian Devices . The solution comes with application and underlying stack conformant to regional standards - WAVE / ETSI and China.
+
+V2X Application Description:    
+==============================
+Following Vehicle To Infrastructure (V2I) applications are supported:
+
+Roadside Alert (RSA): RSU periodically broadcast advisory messages about the road hazards (Slippery Road, Animal On the Road etc.)
```
Comment (amr-mokhtar) [inline applications/altran-v2x/readme.md:13]: Would you like to make as bullet points?
Should add `* ` in front of each element in the list.

File: applications/altran-v2x/readme.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+Altran provided container images for V2X RSU/OBU Apps
+Note: Air interface integration requires third party radio integration for C-V2X PC5 interface and MNO application server for C-V2X Uu interface
+
+
+
+Deploy service:
+=================
+From the Controller Node, go to helm chart folder of the repository:
+
+$ helm install altran <helm-chart location> --set service.type=NodePort
```
Comment (amr-mokhtar) [inline applications/altran-v2x/readme.md:42]: Bound with a code block as below:
\```shell
$ ...
\```

File: applications/altran-v2x/readme.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+Show helm is installed
+
+$helm get all altran
```
Comment (amr-mokhtar) [inline applications/altran-v2x/readme.md:46]: same here

File: applications/altran-v2x/readme.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Altran Technology
+
+Overview of Altran V2X Apps:
+=============================
```
Comment (amr-mokhtar) [inline applications/altran-v2x/readme.md:5]: This is fine, but it is advised to use markdown syntax for headings, such as: #, ##, ###, ...

File: applications/altran-v2x/readme.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+Altran V2X solution enables safety & non-safety applications for V2V,V2I,V2P and V2N scenarios for Road Side Units (RSU), On Board Units (OBU) and Smart Pedestrian Devices . The solution comes with application and underlying stack conformant to regional standards - WAVE / ETSI and China.
+
+V2X Application Description:
```
Comment (amr-mokhtar) [inline applications/altran-v2x/readme.md:9]: No need for the `:` character.

**Review by abjyoti — COMMENTED** [review]
Please resolve the comments

**Review by kuldeep2samasi — COMMENTED** [review]
Taken carereview comments.

File: applications/altran-v2x/readme.md (hunk 1 above)
Comment (abjyoti) [reply applications/altran-v2x/readme.md:31]: These comments are not yet resolved?

File: applications/altran-v2x/readme.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
+```
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2021 Altran Technology
+```
+
+Overview of Altran V2X Apps
+############################
```
Comment (amr-mokhtar) [inline applications/altran-v2x/readme.md:7]: File does not seem to be rendered properly.
https://github.com/open-ness/edgeapps/blob/d8ee8dae6d03a4c3355815b3e6141b27680b6e92/applications/altran-v2x/readme.md

Correct syntax:
```
# Overview of Altran V2X Apps
```

**Review by abjyoti — APPROVED** [review]

**Pull request merged**